makelib/variables.mk

```diff
@@ -16,12 +16,16 @@ FILELISTDIR := $(BUILDDIR)/filelists
 MAINTEMPDIR := $(BUILDDIR)/tmp
 CLEANDIR := $(BUILDDIR)/clean
 
+# host tools
 ACTOOL := $(TOOLSDIR)/actool
 DEPSGENTOOL := $(TOOLSDIR)/depsgen
 FILELISTGENTOOL := $(TOOLSDIR)/filelistgen
 CLEANGENTOOL := $(TOOLSDIR)/cleangen
 QUICKRMTOOL := $(TOOLSDIR)/quickrm
 
+# target tools
+ACTOOL_TARGET := $(TARGET_BINDIR)/actool
```

Comment:
Ah, this change can be actually dropped. We don't distribute actool in tarballs. If somebody wants to have an `actool` binary, then they should just build it from appc/spec.